pkg/egressgateway/manager.go

```diff
@@ -628,6 +628,12 @@ func (manager *Manager) addMissingIpRulesAndRoutes(isRetry bool) (shouldRetry bo
 			logfields.LinkIndex: gwc.ifaceIndex,
 		})
 
+		if err := addEgressIpRoutes(gwc); err != nil {
+			logger.WithError(err).Warn("Can't add IP routes")
+		} else {
+			logger.Debug("Added IP routes")
+		}
+
```

Comment:
btw, I think we should `continue` here on an error (and not try to add any useless IP rules). And maybe even set `shouldRetry`, although I would need to refresh my memory for what cases we expect it to help ...